libethdrivers/CMakeLists.txt

```diff
@@ -118,6 +118,7 @@ string(
     "-Wl"
     "--undefined=tx2_ether_qos_ptr"
     "--undefined=zynq7000_gem_ptr"
+    "--undefined=zynqmp_gem_ptr"
```

Comment:
I don't understand why this is needed for a library. And if it's not a library, then why not discard it if unused?

Reply:
Those symbols get put into a special named section. At runtime the section gets iterated over when looking for compatible drivers. Because the symbols aren't directly linked the object file may not end up being linked in if the symbols aren't explicitly marked undefined.